src/idl_parser.cpp

```diff
@@ -2527,6 +2532,10 @@ CheckedError Parser::ParseDecl() {
   ECHECK(StartStruct(name, &struct_def));
   struct_def->doc_comment = dc;
   struct_def->fixed = fixed;
+  if (filename != nullptr && !opts.project_root.empty()) {
```

Comment:
drop `!= nullptr`

Reply:
Done

Reply:
yes, with so many contributors, things are bound to get a bit random :(